src/components/config-editor/config-editor-header.tsx

```diff
@@ -49,6 +52,7 @@ export function mapDispatchToProps(dispatch: Dispatch<EditorActions.Action>) {
 
 function mapStateToProps(state) {
   return {
+    manualParse: state.manualParse,
```

Comment:
This mapStateToProps shouldn't even be here.

Reply:
Where should I keep it then?

Reply:
we can remove the manual parse but I think we need the themeName in this case

Reply:
Check my commits from today.